app.py

```diff
@@ -49,6 +76,68 @@ def dated_url_for(endpoint, **values):
     return url_for(endpoint, **values)
 
 
+@app.route('/lsmodels/')
```

Comment:
This isn't very RESTful, where paths should be the name of resources, and the HTTP commands should be actions.

So, ideally `GET /models` should list the existing models, `POST /models` (or maybe `PUT /models/HPO`?) should submit a training job, and `DELETE /models/HPO` should delete one model. Can Flask do that easily, route different HTTP methods to different Python methods?

Reply:
I have made revisions, you can now `GET /models/` to see the list of available trained models and you can also `DELETE /models/NameOfModelToDelete` to remove a model. Routing different HTTP methods to different Python methods in Flask is simple. `PUT` / `POST` functionality will be added in the PR containing the model training code.